fabfile.py

```diff
@@ -166,6 +166,58 @@ def create_local_ini():
 
 
 @task
+def migrate_local_ini():
```

Comment:
Is this task used only for development of this .rc system? Or is it meant to be used in a regular pipeline?